frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature

```diff
@@ -0,0 +1,157 @@
+                  | sample-yaml-upload-a | Deployment        |
+                  | sample-yaml-upload-b | Deployment Config |
+
+
+        @regression
+        Scenario: Upload JAR file with advanced option "Health Checks" : A-04-TC12
+             When user is on the Upload JAR file form
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload"
+              And user clicks "Health Checks" link in Advanced Options section
+              And user fills the Readiness Probe details
+              And user fills the Liveness Probe details
+              And user fills the Startup Probe details
+              And user clicks Create button on Add page
+             Then user will be redirected to Topology page
+
+
+        @regression
+        Scenario: Upload JAR file without application route : A-04-TC05
+             When user is on the Upload JAR file form
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload"
+              And user unselects the advanced option Create a route to the application
+              And user clicks Create button on Add page
+             Then user will be redirected to Topology page
+              And public url is not created for node "sample-yaml-upload" in the workload sidebar
+
+
+        @regression
+        Scenario: Upload JAR file with advanced option "Routing" : A-04-TC06
+             When user is on the Upload JAR file form
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload"
+              And user clicks "Routing" link in Advanced Options section
+              And user enters Hostname as "home"
+              And user enters Path as "/home"
+              And user selects default Target Port
+              And user clicks Create button on Add page
+             Then user will be redirected to Topology page
+              And the route of application "sample-yaml-upload" contains "home" in the Routes section of the workload sidebar
+
+
+        @regression
+        Scenario: Upload JAR file by unselecting options in "Build Configuration" section: A-04-TC07
+             When user is on the Upload JAR file form
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload"
+              And user clicks "Build Configuration" link in Advanced Options section
+              And user unselects Configure a webhook build trigger checkbox in build configuration section
```

Comment:
These options aren't available for upload a jar flow.

Reply:
Updated